[21.11] webkitgtk: disable hardware acceleration

blitz opened the pull request:
###### Description of changes

Since the update to webkitgtk 2.36 in #167920, webkitgtk enables hardware acceleration by default. This fully breaks GNOME Evolution for me, because the email compose window becomes unusable.

There is a workaround by setting `WEBKIT_DISABLE_COMPOSITING_MODE=1`, but I assume that other apps that use webkit are also affected. So this PR reverts webkit back to its 2.34 behavior of not using hardware acceleration by default.

Fixes #168645. Relates to #168532.

Marked as draft until my builds complete.

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

@NixOS/gnome

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

blitz committed:
webkitgtk: disable hardware acceleration

Since the update to 2.36, webkitgtk enables hardware acceleration by
default. This causes problems in applications like Evolution on
specific hardware platforms (Intel Tiger Lake), where HTML views
become empty grey panes.

hmenke commented:
Wouldn't a patch like this be much more idiomatic?
```patch
diff --git a/Source/WebKit/UIProcess/gtk/HardwareAccelerationManager.cpp b/Source/WebKit/UIProcess/gtk/HardwareAccelerationManager.cpp
index 8f10c7e6..9c827a4d 100644
--- a/Source/WebKit/UIProcess/gtk/HardwareAccelerationManager.cpp
+++ b/Source/WebKit/UIProcess/gtk/HardwareAccelerationManager.cpp
@@ -38,7 +38,7 @@ HardwareAccelerationManager& HardwareAccelerationManager::singleton()
 }
 
 HardwareAccelerationManager::HardwareAccelerationManager()
-    : m_canUseHardwareAcceleration(true)
+    : m_canUseHardwareAcceleration(false)
     , m_forceHardwareAcceleration(false)
 {
 #if !ENABLE(WEBGL)
```

blitz commented:
> Wouldn't a patch like this be much more idiomatic?

True. I've updated the patch. I modified the `getenv` line earlier, because it seemed easier to review for people unfamiliar with C++.

hmenke reviewed (approved):
Works fine.

![Screenshot from 2022-04-17 18-30-47](https://user-images.githubusercontent.com/1903556/163723697-7527f8b4-d8f7-43d4-b780-8956f3f38e43.png)

hmenke commented:
Result of `nixpkgs-review pr 169058` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>gnome.gnome-documents</li>
    <li>xmonad_log_applet</li>
  </ul>
</details>
<details>
  <summary>209 packages failed to build:</summary>
  <ul>
    <li>ChowKick</li>
    <li>adapta-gtk-theme</li>
    <li>alfis</li>
    <li>almanah</li>
    <li>apostrophe</li>
    <li>astroid</li>
    <li>aws-workspaces</li>
    <li>balsa</li>
    <li>bespokesynth</li>
    <li>birdfont</li>
    <li>bookworm</li>
    <li>bubblemail</li>
    <li>calls</li>
    <li>chatty</li>
    <li>chrome-gnome-shell</li>
    <li>cinnamon.cinnamon-common</li>
    <li>cinnamon.cinnamon-control-center</li>
    <li>cinnamon.cinnamon-gsettings-overrides</li>
    <li>cinnamon.cinnamon-screensaver</li>
    <li>citrix_workspace (citrix_workspace_21_09_0)</li>
    <li>citrix_workspace_20_04_0</li>
    <li>citrix_workspace_20_06_0</li>
    <li>citrix_workspace_20_09_0</li>
    <li>citrix_workspace_20_10_0</li>
    <li>citrix_workspace_20_12_0</li>
    <li>citrix_workspace_21_01_0</li>
    <li>citrix_workspace_21_03_0</li>
    <li>citrix_workspace_21_06_0</li>
    <li>citrix_workspace_21_08_0</li>
    <li>claws-mail</li>
    <li>cog</li>
    <li>dbeaver</li>
    <li>dropbox-cli</li>
    <li>eclipse-mat</li>
    <li>eclipses.eclipse-committers</li>
    <li>eclipses.eclipse-cpp</li>
    <li>eclipses.eclipse-java</li>
    <li>eclipses.eclipse-jee</li>
    <li>eclipses.eclipse-modeling</li>
    <li>eclipses.eclipse-platform</li>
    <li>eclipses.eclipse-rcp</li>
    <li>eclipses.eclipse-scala-sdk</li>
    <li>eclipses.eclipse-sdk</li>
    <li>elementary-planner</li>
    <li>empathy (gnome.empathy)</li>
    <li>eolie</li>
    <li>ephemeral</li>
    <li>epiphany (gnome.epiphany)</li>
    <li>evolution-ews</li>
    <li>evolutionWithPlugins</li>
    <li>feedreader</li>
    <li>foliate</li>
    <li>folks (gnome.folks)</li>
    <li>font-manager</li>
    <li>formiko</li>
    <li>gfbgraph (gnome.gfbgraph)</li>
    <li>giara</li>
    <li>glade (gnome.glade)</li>
    <li>gnome-builder</li>
    <li>gnome-feeds</li>
    <li>gnome-inform7</li>
    <li>gnome-photos (gnome.gnome-photos)</li>
    <li>gnome-recipes</li>
    <li>gnome.bijiben (gnome.gnome-notes)</li>
    <li>gnome.cheese</li>
    <li>gnome.devhelp</li>
    <li>gnome.file-roller</li>
    <li>gnome.geary</li>
    <li>gnome.gnome-applets</li>
    <li>gnome.gnome-books</li>
    <li>gnome.gnome-boxes</li>
    <li>gnome.gnome-calendar</li>
    <li>gnome.gnome-contacts</li>
    <li>gnome.gnome-control-center (gnome.gnome_control_center)</li>
    <li>gnome.gnome-flashback</li>
    <li>gnome.gnome-initial-setup</li>
    <li>gnome.gnome-maps</li>
    <li>gnome.gnome-music</li>
    <li>gnome.gnome-online-miners</li>
    <li>gnome.gnome-panel</li>
    <li>gnome.gnome-session (gnome.gnome_session)</li>
    <li>gnome.gnome-shell (gnome.gnome_shell)</li>
    <li>gnome.gnome-terminal (gnome.gnome_terminal)</li>
    <li>gnome.gnome-todo</li>
    <li>gnome.gnome-tweaks (gnome.gnome-tweak-tool)</li>
    <li>gnome.gnome-user-share</li>
    <li>grilo-plugins (gnome.grilo-plugins)</li>
    <li>gnome.gvfs</li>
    <li>libgepub (gnome.libgepub)</li>
    <li>libzapojit (gnome.libzapojit)</li>
    <li>gnome.nautilus</li>
    <li>gnome.nautilus-python</li>
    <li>gnome.polari</li>
    <li>shotwell (gnome.shotwell)</li>
    <li>gnome.sushi</li>
    <li>gnome.totem</li>
    <li>tracker-miners (gnome.tracker-miners)</li>
    <li>yelp (gnome.yelp)</li>
    <li>gnomeExtensions.easyScreenCast</li>
    <li>gnomeExtensions.gsconnect</li>
    <li>gnomeExtensions.night-theme-switcher</li>
    <li>gnucash</li>
    <li>gnunet-gtk</li>
    <li>gnvim</li>
    <li>gnvim-unwrapped</li>
    <li>gthumb</li>
    <li>haskellPackages.ghcjs-dom-hello</li>
    <li>haskellPackages.gi-javascriptcore</li>
    <li>haskellPackages.gi-webkit2</li>
    <li>haskellPackages.gi-webkit2webextension</li>
    <li>haskellPackages.jsaddle-webkit2gtk</li>
    <li>haskellPackages.reflex-dom</li>
    <li>haskellPackages.reflex-dom-fragment-shader-canvas</li>
    <li>haskellPackages.reflex-localize-dom</li>
    <li>haskellPackages.webkit2gtk3-javascriptcore</li>
    <li>jami-client-gnome</li>
    <li>kgx</li>
    <li>kicad-small</li>
    <li>kicad-unstable-small</li>
    <li>kotatogram-desktop</li>
    <li>liferea</li>
    <li>lispPackages.cl-webkit2</li>
    <li>lispPackages.nyxt</li>
    <li>luakit</li>
    <li>lutris</li>
    <li>lutris-free</li>
    <li>lutris-unwrapped</li>
    <li>mailnagWithPlugins</li>
    <li>marker</li>
    <li>mate.atril</li>
    <li>mate.mate-user-guide</li>
    <li>mavproxy</li>
    <li>meteo</li>
    <li>midori-unwrapped</li>
    <li>mmex</li>
    <li>nasc</li>
    <li>newsflash</li>
    <li>notes-up</li>
    <li>nyxt</li>
    <li>onboard</li>
    <li>osmo</li>
    <li>pantheon.elementary-calendar</li>
    <li>pantheon.elementary-capnet-assist</li>
    <li>pantheon.elementary-code</li>
    <li>pantheon.elementary-greeter</li>
    <li>pantheon.elementary-gsettings-schemas</li>
    <li>pantheon.elementary-mail</li>
    <li>pantheon.elementary-photos</li>
    <li>pantheon.elementary-session-settings</li>
    <li>pantheon.elementary-tasks</li>
    <li>pantheon.epiphany</li>
    <li>pantheon.extra-elementary-contracts</li>
    <li>pantheon.file-roller</li>
    <li>pantheon.notes-up</li>
    <li>pantheon.switchboard-plug-a11y</li>
    <li>pantheon.switchboard-plug-keyboard</li>
    <li>pantheon.switchboard-plug-onlineaccounts</li>
    <li>pantheon.switchboard-with-plugs</li>
    <li>pantheon.wingpanel-applications-menu</li>
    <li>pantheon.wingpanel-indicator-datetime</li>
    <li>pantheon.wingpanel-with-indicators</li>
    <li>pdfpc</li>
    <li>phosh</li>
    <li>playonlinux</li>
    <li>poedit</li>
    <li>portfolio</li>
    <li>printrun</li>
    <li>protonvpn-gui</li>
    <li>python38Packages.humblewx</li>
    <li>python38Packages.kicad</li>
    <li>python38Packages.wxPython_4_0</li>
    <li>python38Packages.wxPython_4_1</li>
    <li>python39Packages.humblewx</li>
    <li>python39Packages.kicad</li>
    <li>python39Packages.wxPython_4_0</li>
    <li>python39Packages.wxPython_4_1</li>
    <li>pytrainer</li>
    <li>quisk</li>
    <li>quodlibet</li>
    <li>quodlibet-full</li>
    <li>quodlibet-without-gst-plugins</li>
    <li>quodlibet-xine</li>
    <li>quodlibet-xine-full</li>
    <li>rednotebook</li>
    <li>remmina</li>
    <li>rymcast</li>
    <li>setzer</li>
    <li>skytemple</li>
    <li>sparkleshare</li>
    <li>surf</li>
    <li>surf-display</li>
    <li>synology-drive</li>
    <li>tangram</li>
    <li>tdesktop</li>
    <li>thiefmd</li>
    <li>timeline</li>
    <li>tonelib-zoom</li>
    <li>tunefish</li>
    <li>ulauncher</li>
    <li>vala-language-server</li>
    <li>vimb-unwrapped</li>
    <li>vimix-gtk-themes</li>
    <li>vocal</li>
    <li>webkit2-sharp</li>
    <li>webkitgtk_4_1</li>
    <li>whatsapp-for-linux</li>
    <li>whitesur-gtk-theme</li>
    <li>wike</li>
    <li>xiphos</li>
  </ul>
</details>
<details>
  <summary>5 packages built:</summary>
  <ul>
    <li>evolution</li>
    <li>evolution-data-server (gnome.evolution-data-server ,gnome.evolution_data_server)</li>
    <li>gnome-online-accounts (gnome.gnome-online-accounts ,gnome.gnome_online_accounts)</li>
    <li>libgdata (gnome.libgdata)</li>
    <li>webkitgtk (gnome.webkitgtk)</li>
  </ul>
</details>

hmenke commented:
Hm, that's weird.
```
error: attribute 'bijiben' missing                                                                                     
                                                                                                                       
       at /home/henri/.cache/nixpkgs-review/pr-169058-1/build.nix:71:9:                                             
                                                                                                                       
           70|         gnome-recipes                                                                                   
           71|         gnome."bijiben"                                                                                 
             |         ^                                                                                               
           72|         gnome."cheese"
```
Anyway, not related to this PR.

hmenke commented:
Result of `nixpkgs-review pr 169058` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>gnome.gnome-documents</li>
    <li>xmonad_log_applet</li>
  </ul>
</details>
<details>
  <summary>16 packages failed to build:</summary>
  <ul>
    <li>citrix_workspace (citrix_workspace_21_09_0)</li>
    <li>citrix_workspace_20_04_0</li>
    <li>citrix_workspace_20_06_0</li>
    <li>citrix_workspace_20_09_0</li>
    <li>citrix_workspace_20_10_0</li>
    <li>citrix_workspace_20_12_0</li>
    <li>citrix_workspace_21_01_0</li>
    <li>citrix_workspace_21_03_0</li>
    <li>citrix_workspace_21_06_0</li>
    <li>citrix_workspace_21_08_0</li>
    <li>quodlibet</li>
    <li>quodlibet-full</li>
    <li>quodlibet-without-gst-plugins</li>
    <li>quodlibet-xine</li>
    <li>quodlibet-xine-full</li>
    <li>xiphos</li>
  </ul>
</details>
<details>
  <summary>198 packages built:</summary>
  <ul>
    <li>ChowKick</li>
    <li>adapta-gtk-theme</li>
    <li>alfis</li>
    <li>almanah</li>
    <li>apostrophe</li>
    <li>astroid</li>
    <li>aws-workspaces</li>
    <li>balsa</li>
    <li>bespokesynth</li>
    <li>birdfont</li>
    <li>bookworm</li>
    <li>bubblemail</li>
    <li>calls</li>
    <li>chatty</li>
    <li>chrome-gnome-shell</li>
    <li>cinnamon.cinnamon-common</li>
    <li>cinnamon.cinnamon-control-center</li>
    <li>cinnamon.cinnamon-gsettings-overrides</li>
    <li>cinnamon.cinnamon-screensaver</li>
    <li>claws-mail</li>
    <li>cog</li>
    <li>dbeaver</li>
    <li>dropbox-cli</li>
    <li>eclipse-mat</li>
    <li>eclipses.eclipse-committers</li>
    <li>eclipses.eclipse-cpp</li>
    <li>eclipses.eclipse-java</li>
    <li>eclipses.eclipse-jee</li>
    <li>eclipses.eclipse-modeling</li>
    <li>eclipses.eclipse-platform</li>
    <li>eclipses.eclipse-rcp</li>
    <li>eclipses.eclipse-scala-sdk</li>
    <li>eclipses.eclipse-sdk</li>
    <li>elementary-planner</li>
    <li>empathy (gnome.empathy)</li>
    <li>eolie</li>
    <li>ephemeral</li>
    <li>epiphany (gnome.epiphany)</li>
    <li>evolution</li>
    <li>evolution-data-server (gnome.evolution-data-server ,gnome.evolution_data_server)</li>
    <li>evolution-ews</li>
    <li>evolutionWithPlugins</li>
    <li>feedreader</li>
    <li>foliate</li>
    <li>folks (gnome.folks)</li>
    <li>font-manager</li>
    <li>formiko</li>
    <li>gfbgraph (gnome.gfbgraph)</li>
    <li>giara</li>
    <li>glade (gnome.glade)</li>
    <li>gnome-builder</li>
    <li>gnome-feeds</li>
    <li>gnome-inform7</li>
    <li>gnome-online-accounts (gnome.gnome-online-accounts ,gnome.gnome_online_accounts)</li>
    <li>gnome-photos (gnome.gnome-photos)</li>
    <li>gnome-recipes</li>
    <li>gnome.bijiben (gnome.gnome-notes)</li>
    <li>gnome.cheese</li>
    <li>gnome.devhelp</li>
    <li>gnome.file-roller</li>
    <li>gnome.geary</li>
    <li>gnome.gnome-applets</li>
    <li>gnome.gnome-books</li>
    <li>gnome.gnome-boxes</li>
    <li>gnome.gnome-calendar</li>
    <li>gnome.gnome-contacts</li>
    <li>gnome.gnome-control-center (gnome.gnome_control_center)</li>
    <li>gnome.gnome-flashback</li>
    <li>gnome.gnome-initial-setup</li>
    <li>gnome.gnome-maps</li>
    <li>gnome.gnome-music</li>
    <li>gnome.gnome-online-miners</li>
    <li>gnome.gnome-panel</li>
    <li>gnome.gnome-session (gnome.gnome_session)</li>
    <li>gnome.gnome-shell (gnome.gnome_shell)</li>
    <li>gnome.gnome-terminal (gnome.gnome_terminal)</li>
    <li>gnome.gnome-todo</li>
    <li>gnome.gnome-tweaks (gnome.gnome-tweak-tool)</li>
    <li>gnome.gnome-user-share</li>
    <li>grilo-plugins (gnome.grilo-plugins)</li>
    <li>gnome.gvfs</li>
    <li>libgdata (gnome.libgdata)</li>
    <li>libgepub (gnome.libgepub)</li>
    <li>libzapojit (gnome.libzapojit)</li>
    <li>gnome.nautilus</li>
    <li>gnome.nautilus-python</li>
    <li>gnome.polari</li>
    <li>shotwell (gnome.shotwell)</li>
    <li>gnome.sushi</li>
    <li>gnome.totem</li>
    <li>tracker-miners (gnome.tracker-miners)</li>
    <li>webkitgtk (gnome.webkitgtk)</li>
    <li>yelp (gnome.yelp)</li>
    <li>gnomeExtensions.easyScreenCast</li>
    <li>gnomeExtensions.gsconnect</li>
    <li>gnomeExtensions.night-theme-switcher</li>
    <li>gnucash</li>
    <li>gnunet-gtk</li>
    <li>gnvim</li>
    <li>gnvim-unwrapped</li>
    <li>gthumb</li>
    <li>haskellPackages.ghcjs-dom-hello</li>
    <li>haskellPackages.gi-javascriptcore</li>
    <li>haskellPackages.gi-webkit2</li>
    <li>haskellPackages.gi-webkit2webextension</li>
    <li>haskellPackages.jsaddle-webkit2gtk</li>
    <li>haskellPackages.reflex-dom</li>
    <li>haskellPackages.reflex-dom-fragment-shader-canvas</li>
    <li>haskellPackages.reflex-localize-dom</li>
    <li>haskellPackages.webkit2gtk3-javascriptcore</li>
    <li>jami-client-gnome</li>
    <li>kgx</li>
    <li>kicad-small</li>
    <li>kicad-unstable-small</li>
    <li>kotatogram-desktop</li>
    <li>liferea</li>
    <li>lispPackages.cl-webkit2</li>
    <li>lispPackages.nyxt</li>
    <li>luakit</li>
    <li>lutris</li>
    <li>lutris-free</li>
    <li>lutris-unwrapped</li>
    <li>mailnagWithPlugins</li>
    <li>marker</li>
    <li>mate.atril</li>
    <li>mate.mate-user-guide</li>
    <li>mavproxy</li>
    <li>meteo</li>
    <li>midori-unwrapped</li>
    <li>mmex</li>
    <li>nasc</li>
    <li>newsflash</li>
    <li>notes-up</li>
    <li>nyxt</li>
    <li>onboard</li>
    <li>osmo</li>
    <li>pantheon.elementary-calendar</li>
    <li>pantheon.elementary-capnet-assist</li>
    <li>pantheon.elementary-code</li>
    <li>pantheon.elementary-greeter</li>
    <li>pantheon.elementary-gsettings-schemas</li>
    <li>pantheon.elementary-mail</li>
    <li>pantheon.elementary-photos</li>
    <li>pantheon.elementary-session-settings</li>
    <li>pantheon.elementary-tasks</li>
    <li>pantheon.epiphany</li>
    <li>pantheon.extra-elementary-contracts</li>
    <li>pantheon.file-roller</li>
    <li>pantheon.notes-up</li>
    <li>pantheon.switchboard-plug-a11y</li>
    <li>pantheon.switchboard-plug-keyboard</li>
    <li>pantheon.switchboard-plug-onlineaccounts</li>
    <li>pantheon.switchboard-with-plugs</li>
    <li>pantheon.wingpanel-applications-menu</li>
    <li>pantheon.wingpanel-indicator-datetime</li>
    <li>pantheon.wingpanel-with-indicators</li>
    <li>pdfpc</li>
    <li>phosh</li>
    <li>playonlinux</li>
    <li>poedit</li>
    <li>portfolio</li>
    <li>printrun</li>
    <li>protonvpn-gui</li>
    <li>python38Packages.humblewx</li>
    <li>python38Packages.kicad</li>
    <li>python38Packages.wxPython_4_0</li>
    <li>python38Packages.wxPython_4_1</li>
    <li>python39Packages.humblewx</li>
    <li>python39Packages.kicad</li>
    <li>python39Packages.wxPython_4_0</li>
    <li>python39Packages.wxPython_4_1</li>
    <li>pytrainer</li>
    <li>quisk</li>
    <li>rednotebook</li>
    <li>remmina</li>
    <li>rymcast</li>
    <li>setzer</li>
    <li>skytemple</li>
    <li>sparkleshare</li>
    <li>surf</li>
    <li>surf-display</li>
    <li>synology-drive</li>
    <li>tangram</li>
    <li>tdesktop</li>
    <li>thiefmd</li>
    <li>timeline</li>
    <li>tonelib-zoom</li>
    <li>tunefish</li>
    <li>ulauncher</li>
    <li>vala-language-server</li>
    <li>vimb-unwrapped</li>
    <li>vimix-gtk-themes</li>
    <li>vocal</li>
    <li>webkit2-sharp</li>
    <li>webkitgtk_4_1</li>
    <li>whatsapp-for-linux</li>
    <li>whitesur-gtk-theme</li>
    <li>wike</li>
  </ul>
</details>

jtojnar commented:
I am pretty sure hardware acceleration was always on on supported hardware, hence the various bug reports about it. Since disabling it will likely have significant impact on performance e.g. for people using Epiphany web browser, I would rather go with some less invasive method.

blitz commented:
> I am pretty sure hardware acceleration was always on on supported hardware, hence the various bug reports about it. Since disabling it will likely have significant impact on performance e.g. for people using Epiphany web browser, I would rather go with some less invasive method.

Fair point. What about wrapping evolution with `WEBKIT_DISABLE_COMPOSITING_MODE=1`?

blitz commented:
Closing in favor of #169170.
